### PR TITLE
Demo app

### PR DESCRIPTION
**Pull request by AWolf81**
I've added a simple demo to test your directive and I'd like to add it for others to preview your directive.

After merging you only need to modify the path in the readme to point to the demo. (Couldn't get it to work with relative path.)

This will fix issue #2

### CODE REVIEW

**Comment by scottcorgan** [comment]
That's awesome! Can you change the directory to examples?

**Comment by AWolf81** [comment]
Sure. 
Should I also create a subdirectory in examples? Something like examples/simple-demo?

**Comment by scottcorgan** [comment]
Sounds great!

File: README.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ angular-gist
 
 AngularJS directive for embedding Github gists.
 
+## Demo
+Please have a look at the [demo](http://htmlpreview.github.io/?https://raw.githubusercontent.com/AWolf81/angular-gist/demo-app/examples/simple-demo/index.html) for a live example.
```
Comment (scottcorgan) [inline README.md]: @AWolf81 looks like we can just use a link this instead: https://github.com/scottcorgan/angular-gist/blob/master/examples/simple-demo/index.html

Comment (scottcorgan) [inline README.md]: Once that change is made, I'll merge! Great job!

Comment (AWolf81) [inline README.md]: No problem. Yes, you're right that should work. I'll add it.

**Comment by scottcorgan** [comment]
@AWolf81 is this good to merge now?

**Comment by AWolf81** [comment]
Yes, it's OK. 

Please check after merging that the readme link is working. 
I can't test it because the link is only working after merging.

**Comment by scottcorgan** [comment]
👍

**Pull request merged**